src/test/java/BaseTest/TestRunner.java

```diff
@@ -0,0 +1,51 @@
+package BaseTest;
+
+import Browser.BrowserFactory;
+import Browser.BrowserСhoice;
+import Steps.MainPage;
+import UserDao.UserRepo;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+
+public class TestRunner {
+    final static String baseUrl = "https://www.citrus.com.ua/uk/";
+
+    public static MainPage mainPage;
+
+    public void loadApplication() {
```

Comment:
I don't see any profit of having this method. Please move all internals into `setUpDriver()`